lib/plug/adapters/cowboy.ex

```diff
@@ -117,6 +117,17 @@ defmodule Plug.Adapters.Cowboy do
     :ranch.child_spec(ref, nb_acceptors, ranch_module, trans_opts, :cowboy_protocol, proto_opts)
   end
 
+  @doc """
+  Returns the current configuration for the listener `ref` as a keyword list. As
```

Comment:
Keep the first line in docs short and in its own paragraph as it is shown as a summary.